### DISCUSSION TITLE
Keyboard and mouse in vertical mode.

### DISCUSSION THREAD

**Discussion by Cesar-AVF** [AppGround-io/bluetooth-keyboard-and-mouse-support#312] [Ideas]
Hello! I don't know if there is an option to configure the interface in a more free way, but I would like it if the mouse could be integrated into both sides of the keyboard in vertical mode.
![Keyboard and Mouse](https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/assets/114028591/fed0f101-d022-4303-a226-8361c73ecaba)

**Comment by AppGround-io** [comment]
Hi,

Thanks for reaching out with your suggestion!

We're always looking for ways to improve the app, and I'll certainly take
your feedback into consideration for future updates. Unfortunately, we
don't have that feature available at the moment, but I'll make a note of it
for the development roadmap.

If you have any further ideas or need assistance with anything else, please
don't hesitate to get in touch.

Best,
Sebastian

**Comment by Dotatay** [comment]
Very cool idea. A custom layout option would also be great.
